### PR TITLE
Add clarity for OpenShift deployments

### PR DESCRIPTION
**Pull request by sdodsley**
Ensure it is obvious you must change the `orchestrator` from the default of `k8s` to `openshift`

### CODE REVIEW

**Review by haibinxie — APPROVED** [review]
LGTM

File: docs/openshift_mc.md [hunk 1]
```diff
@@ -85,3 +85,16 @@ devices {
 /etc/udev/rules.d/99-pure-storage.rules
 /etc/udev/rules.d/90-scsi-ua.rules
 ``` 
+
+## Configure PSO values file for OpenShift
+
+To ensure that when installing PSO on an OpenShift cluster all appropriate OpenShift artifacts, such as SCCs, are correctly creadted you **MUST** edit the Helm `values.yaml`
+configuration file.
+
+Ensure that the `orchestrsator` `name` parameter is set to `openshift` as shown below:
```
Comment (Pure-AdamuKaapan) [inline docs/openshift_mc.md:94]:
```suggestion
Ensure that the `orchestrator` `name` parameter is set to `openshift` as shown below:
```

**Review by Pure-AdamuKaapan — CHANGES_REQUESTED** [review]

**Review by dsupure — APPROVED** [review]
Looks good as long as Adam's comment is addressed.

**Review by Pure-AdamuKaapan — APPROVED** [review]
LGTM!

**Pull request merged**